apps/definition_extract/lib/extract.ex

```diff
@@ -8,22 +8,20 @@ defmodule Extract do
           id: uuid,
           dataset_id: String.t(),
           subset_id: String.t(),
-          destination: String.t(),
-          steps: [Extract.Step.t()],
-          dictionary: Dictionary.t(),
-          message_key: list,
-          config: map
+          source: Source.t(),
+          decoder: Decoder.t(),
+          destination: Destination.t(),
+          dictionary: Dictionary.t()
         }
 
   defstruct version: 1,
```

Comment:
Should this now be version 2?

Reply:
since no one is using hindsight yet and our "prod" environment is technically our dev environment while our dev environment serves as our perf testing environment, I vote we kick this can and keep v1 as long as we don't have any deployments that have state we need to persist from one refactor to the next.